src/stratis_cli/_error_reporting.py

```diff
@@ -94,7 +114,7 @@ def _interpret_errors(errors):
     # pylint: disable=fixme
     # FIXME: remove no coverage pragma when adequate testing for CLI output
     # exists.
-    try:  # pragma: no cover
+    try:
```

Comment:
You can remove the three comment lines above here, which are all an explanation for the removed "no cover" pragma.

Reply:
Oh, and you can remove the "no cover" pragma on the ```_interface_name...``` method as well.